app/src/main/java/com/tari/android/wallet/ui/activity/home/HomeActivity.kt

```diff
@@ -75,6 +75,9 @@ import java.math.BigDecimal
  *
  * @author The Tari Development Team
  */
+
+const val REQUEST_QR_SCANNER = 101
```

Comment:
Not sure if needs to be an app-wide constant, can be in the class intead. If it has to be a global, maybe it's better if it resides in the QR activity file.